lib/Sema/TypeCheckDecl.cpp

```diff
@@ -1418,7 +1402,7 @@ void TypeChecker::computeAccessibility(ValueDecl *D) {
 namespace {
 
 class TypeAccessScopeDiagnoser : private ASTWalker {
-  const DeclContext *accessScope;
+  const AccessScope accessScope;
```

Comment:
No need for `const` here either, although I suppose it does make a difference this time.